### ISSUE TITLE
How to append TraceId to Message field to use Loki Derived fields in Grafana

### ISSUE THREAD

**Issue by Mario-Hofstaetter** [mishamyte/serilog-sinks-grafana-loki#7]
First thank you for this sink, its working very nicely in my dev setup so far.

I have more of an question than an issue.
I am using https://github.com/open-telemetry/opentelemetry-dotnet and `jaeger` for tracing. Since this adds a boatload of properties with very high cardinality (`TraceId`), the `LokiLabelFiltrationMode.Include` rescued loki from dying.

But to correlate from logs (loki) to traces (jaeger) within grafana that `TraceId` is needed. This works especially nice with [Derived fields](https://grafana.com/docs/grafana/latest/datasources/loki/#derived-fields) in grafana explore.

For this to work the `TraceId` is needed in the `{Message}` data that loki stores.

**Is there a way to automatically format / append the `TraceId` Property to the `Message` when using your loki sink?**

I`d like to avoid this manual approach:
```csharp
_logger.LogError("Test Derived fields in loki with manual TraceId={TraceId}", Activity.Current?.TraceId);
```

`outputTemplate` used for `Console` sink: 
```csharp
@"[{Timestamp:o} {Level:u3}] {TraceId:l} {Message:lj}{NewLine}{Exception}"
```

The result for any readers:
![grafik](https://user-images.githubusercontent.com/33002073/99156985-b7438380-26c5-11eb-98de-7e16ab5be311.png)

![grafik](https://user-images.githubusercontent.com/33002073/99156989-bd396480-26c5-11eb-9d35-e0aa80082b4f.png)

**Comment by mishamyte** [comment]
Hello, Mario!

Thanks for your appreciation.

Yes, seems like there is a bug & formatter is missed while generating entry.
I've done some draft fixes, will test them and will publish the fix in a few days

**Comment by mishamyte** [comment]
@Mario-Hofstaetter behavior was fixed in [v4.0.1](https://github.com/mishamyte/serilog-sinks-grafana-loki/releases/tag/v4.0.1).
Please, try and let me know how it is

**Comment by Mario-Hofstaetter** [comment]
@mishamyte it seems to be working fine now in `4.0.1` , thanks for the quick fix.

![grafik](https://user-images.githubusercontent.com/33002073/99160083-8c682800-26e3-11eb-95e3-481b73dba68a.png)

To anybody google landing here, the working code (only development setup for me atm):

```csharp
public const string LOKI_LOG_TEMPLATE = @"({TraceId:l}) {Message:lj} {Exception}";

public static int Main(string[] args)
{
	Log.Logger = new LoggerConfiguration()
		.MinimumLevel.Debug()
		.MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
		.WriteTo.Console()
		.WriteTo.GrafanaLoki("http://localhost:3100", 
		    filtrationLabels: new []
			{
				"Service",
				"Resource",
				"SourceContext", 
				"Method"
			}, 
			filtrationMode:LokiLabelFiltrationMode.Include,
			labels: new[] {new LokiLabel {Key = "Service", Value = Startup.Source.Name}},
			outputTemplate: LOKI_LOG_TEMPLATE)
		.CreateLogger();

// [...]
```

Rather generic regex if I happen to change the formatting:

![grafik](https://user-images.githubusercontent.com/33002073/99160104-e36dfd00-26e3-11eb-8353-3b46dc847fdf.png)
